Client/components/MessageForm.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import '../public/style.css';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
+import ls from 'local-storage';
+
+class MessageForm extends Component {
+
+submit(event){
+  event.preventDefault();
+
+  var currentdate = new Date(); 
+  var datetime = currentdate.getDate() + "/"
```

Comment:
Be consistent, in some files i see `let`, here i see `var`
If it's es6, stick to the es6 conventions for declaration, variable naming, function style